### DISCUSSION TITLE
Keeping empyrical alive by bundling it with EigenLedger

### DISCUSSION THREAD

**Discussion by santoshlite** [ssantoshp/Empyrial#128] [Announcements]
Hey everyone,

I've been thinking a lot about [`empyrical`](https://github.com/quantopian/empyrical) lately. As you know, EigenLedger relies heavily on it, but it's becoming increasingly buggy with newer Python versions since it's no longer maintained (after Quantopian got acquired by Robinhood).

So here's my idea: I'm planning to bundle empyrical with EigenLedger and take on active maintenance. This means I'll be fixing bugs, ensuring compatibility with new Python releases, and actively merging PRs from anyone who wants to contribute. By doing this, we can keep `empyrical` alive and well for all of us who depend on it.

As always, everyone's welcome to collaborate! Feel free to open PRs, share feedback, or contribute in any way you can. Let's keep this great tool running smoothly together.

You can check out how to use `empyrical` from EigenLedger here in the meantime: https://eigenledger.gitbook.io/eigenledger/using-empyrical/using-empyrical

Cheers :)

**Comment by kartiksubbarao** [comment]
FYI you might want to check out https://github.com/stefan-jansen/empyrical-reloaded which @stefan-jansen has been maintaining, it has several bugfixes applied.
